helm_deploy/values-preprod.yaml

```diff
@@ -11,7 +11,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     HMPPS_AUTH_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
     SERVICES_COMMUNITY-API_BASE-URL: https://community-api.pre-prod.delius.probation.hmpps.dsd.io
-    SERVICES_AP-DELIUS-CONTEXT-API_BASE-URL: https://approved-premises-and-delius.pre-prod.delius.probation.hmpps.dsd.io
+    SERVICES_AP-DELIUS-CONTEXT-API_BASE-URL: https://approved-premises-and-delius-preprod.hmpps.service.justice.gov.uk
```

Comment:
Just to point out that I'm getting a certificate warning from this host:

```
curl "https://approved-premises-and-delius-preprod.hmpps.service.justice.gov.uk/approved-premises/Q057/staff?size=10"
curl: (60) SSL: no alternative certificate subject name matches target host name 'approved-premises-and-delius-preprod.hmpps.service.justice.gov.uk'
More details here: https://curl.se/docs/sslcerts.html

```

Is that expected?

Reply:
Now responding without complaint, thks!